docs/_Research-Data-Management/13-data-qc.md

```diff
@@ -1,196 +1,387 @@
-4. almost no gene is DE
-- **source**: DESeq2, negative control study
-- **possible reason(s)**: negative control study (FP)
-
-### Visualize
-1. using CPM to visualize DEGs
-- **source**: Plot, negative control study
-- **possible reason(s)**: mixed up within- and between-sample normalization
-- **solution/measure**: use correct normalization
-2. use raw Ratios to visualize DEGs effect size
-- **source**: Plot, negative control study
-- **possible reason(s)**: humans are bad with ratios (0.01 = almost 0 and 100 is just large but not the largest bar ever)
-- **solution/measure**: use any log transformation (e.g. log10: 0.01 => -2, 100 => +2)
-
-# Single cell
-
-### Quality check
-1. peak at left/right side in gene or reads per cell histogram or log10-cummulative-number of reads per cell id
-- **source**: BD rhapsody pipeline, negative control study
-- **possible reason(s)**: left=cell fragments, right=multiplets present
-- **solution/measure**: remove with cut-offs
-2. inflated UMI counts
-- **source**: BD rhapsody pipeline, negative control study
-- **possible reason(s)**: using raw UMI counts
-- **solution/measure**: use DBEC/RSEC UMI counts
-
-### Dim. reduction
-1. poor PCA/UMAP/tSNE embedding
-- **source**: Dim. reduction embedding, negative control study
-- **possible reason(s)**: using e.g. only one assay of count data for embedding
-- **solution/measure**: for multimodal data use a WNN approach (combining both assays)
-2. poor PCA/UMAP/tSNE embedding
-- **source**: Dim. reduction embedding, negative control study
-- **possible reason(s)**: use raw data for tSNE/UMAP
-- **solution/measure**: use a significant portion of PC from the PCA as input for tSNE/UMAP
-3. "The first two principle components were used to perform a tSNE" <https://doi.org/10.1126/science.aag3009>
-- **source**: Dim. reduction embedding, negative control study
-- **possible reason(s)**: use only 2 PC from the PCA for the tSNE/UMAP projection
-- **solution/measure**: use a significant portion of PC from the PCA as input for tSNE/UMAP
-
-### Find subpopulations
-1. cluster form based for a specific batch index
-- **source**: Dim. reduction embedding + clustering, negative control study
-- **possible reason(s)**: batch effect
-- **solution/measure**: correct for batch effect (e.g. integrate using seurat)
-
-### DEA
-1. many DEGs
-- **source**: seurat/deseq, negative control study
-- **possible reason(s)**: DEG between very small sub-populations
-- **solution/measure**: use a population size cutoff or state the number
-2. some genes look like dates (1-Mar,...)
-- **source**: seurat/deseq, negative control study
-- **possible reason(s)**: some genes can be interpreted as dates when using excel for data handling <https://doi.org/10.1126/science.aah4573>
-- **solution/measure**: never ever use excel or at least make sure that cell type is not "AUTO"
-
-# Further resources
-
-# References
+
+
+### 🚩 Library contamination or highly homogenous library 
```

Comment:
Call me a purist and I know this is a personal opinion but titles should not have emojis. A more rational argument is that this could mess up automated TOC or the navigation with inclusive browsers (using only tabs etc).